bindings/rust/wasmedge-sdk/src/externals/function.rs

```diff
@@ -141,7 +146,7 @@ impl Func {
     }
 }
 
-/// Defines a type builder for creating a [FuncType](wasmedge_types::FuncType).
+/// Defines a type builder for creating a [FuncType](https://wasmedge.github.io/WasmEdge/wasmedge_types/struct.FuncType.html) instance.
```

Comment:
I would like to know when to use the crate link or URL?

Reply:
If the content is in the current crate, use `crate::xxx`; otherwise, use `URL`. In this PR, for example, `FuncType` is defined in the `wasmedge-types` crate, so have to use `URL`. Thanks!

Reply:
Got it.